composable_kernel/include/tensor_operation/element_wise_operation.hpp

```diff
@@ -12,6 +12,31 @@ struct PassThrough
     __host__ __device__ void operator()(half_t& y, const half_t& x) const { y = x; }
 };
 
+struct Add
+{
+    template <typename T1, typename T2, typename T3>
+    __host__ __device__ constexpr void operator()(T1& dst, const T2& src_y, const T3& bias) const
+    {
+        dst = static_cast<T1>(src_y + static_cast<T2>(bias));
+    }
+};
+
+struct AlphaBetaAdd
+{
+    AlphaBetaAdd(float alpha, float beta) : alpha_(alpha), beta_(beta) {}
+
+    template <typename T1, typename T2, typename T3>
+    __host__ __device__ constexpr void operator()(T1& dst, const T2& src_y, const T3& bias) const
```

Comment:
Let's not use templated type, but real type here

Reply:
Fix
https://github.com/ROCmSoftwarePlatform/composable_kernel/commit/9bafd938f7cf09a809c0183074cf30f8664cd93b#diff-990561d35152da39a727f324616ba178793b8b1e20ecd17194b48988282cbd66R36-R47